browser/tor/sources.gni

```diff
@@ -0,0 +1,43 @@
+# Copyright (c) 2020 The Brave Authors. All rights reserved.
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this file,
+# You can obtain one at http://mozilla.org/MPL/2.0/.
+#
+
+import("//brave/browser/translate/buildflags/buildflags.gni")
+import("//brave/components/tor/buildflags/buildflags.gni")
+
+brave_browser_tor_sources = []
+
+brave_browser_tor_deps = []
+
+if (enable_tor) {
+  brave_browser_tor_sources += [
+    "//brave/browser/tor/onion_location_navigation_throttle_delegate.cc",
+    "//brave/browser/tor/onion_location_navigation_throttle_delegate.h",
+    "//brave/browser/tor/tor_profile_manager.cc",
+    "//brave/browser/tor/tor_profile_manager.h",
+    "//brave/browser/tor/tor_profile_service_factory.cc",
+    "//brave/browser/tor/tor_profile_service_factory.h",
+  ]
+
+  brave_browser_tor_deps += [
+    "//base",
+    "//brave/browser:browser_process",
+    "//brave/browser/translate/buildflags",
+    "//brave/common",
+    "//brave/components/tor",
+    "//chrome/browser:browser_process",
+    "//chrome/browser/ui",
+    "//chrome/common",
+    "//components/keyed_service/content",
+    "//components/prefs",
+    "//components/safe_browsing/core/common",
+    "//content/public/browser",
+    "//third_party/blink/public/common",
+  ]
+
+  if (enable_brave_translate_extension) {
+    brave_browser_tor_deps += [ "//components/translate/core/browser" ]
```

Comment:
it seems we can drop this, since there is nothing meaningful in that file

Reply:
not sure what you mean, tor_profile_manager.cc has
```
#if BUILDFLAG(ENABLE_BRAVE_TRANSLATE_EXTENSION)
  pref_service->SetBoolean(prefs::kOfferTranslateEnabled, false);
#endif
```
so we can't drop this, but I can make it more specific with `"//components/translate/core/browser:translate_pref_names",`

Reply:
ah, right, I've got it confused with `//brave/components/translate`